src/frontend/src/pages/DRCPage/DRCPage.tsx

```diff
@@ -0,0 +1,141 @@
+    }
+  ]);
+  console.log(EventDict);
+  const handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleDropdownClose = () => {
+    setAnchorEl(null);
+  };
+
+  const daysInMonth = (month: Date) => {
+    return new Date(month.getFullYear(), month.getMonth() + 1, 0).getDate();
+  };
+
+  const paddingDays = (month: Date) => {
+    return new Date(month.getFullYear(), month.getMonth(), 0).getDay();
+  };
+
+  const paddingArrayStart = [...Array<number>(paddingDays(displayMonth)).keys()].map(
+    (day) => daysInMonth(new Date(displayMonth.getDate(), displayMonth.getMonth() - 1, displayMonth.getFullYear())) - day
+  );
+  const paddingArrayEnd = [...Array<number>(7 - ((daysInMonth(displayMonth) + paddingDays(displayMonth)) % 7)).keys()].map(
+    (day) => day + 1
+  );
+  const daysThisMonth = paddingArrayStart
+    .concat([...Array(daysInMonth(displayMonth)).keys()])
+    .map((day) => day + 1)
+    .concat(paddingArrayEnd.length < 7 ? paddingArrayEnd : []);
+
+  const unconfirmedDRSDropdown = (
+    <>
+      <NERButton
+        endIcon={<ArrowDropDownIcon style={{ fontSize: 28 }} />}
+        variant="contained"
+        id="unconfirmed-drs-dropdown"
+        onClick={handleClick}
+      >
+        My Unconfirmed DRS
+      </NERButton>
+      <Menu open={!!anchorEl} anchorEl={anchorEl} onClose={handleDropdownClose}>
+        <MenuItem
+          onClick={() => {
+            return;
+          }}
+        >
+          <ListItemIcon>
+            <NoteAddIcon fontSize="small" />
+          </ListItemIcon>
+          Mock Data
+        </MenuItem>
+      </Menu>
+    </>
+  );
+
+  return (
+    <PageLayout title="Design Review Calendar" headerRight={unconfirmedDRSDropdown}>
+      <MonthSelector
+        displayMonth={displayMonth}
+        setDisplayMonth={(date) => {
+          setDisplayMonth(date);
+        }}
+      ></MonthSelector>
+      <Grid container alignItems="center">
+        {['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday'].map((day) => (
+          <Grid item xs={12 / 7}>
+            <Typography align={'center'} sx={{ fontWeight: 'bold', fontSize: 18 }}>
+              {day}
+            </Typography>
+          </Grid>
+        ))}
+      </Grid>
+      <Box sx={{ border: '2px solid grey', borderRadius: 2, bgcolor: theme.palette.background.paper }}>
+        <Grid container>
+          {[0, 7, 14, 21, 28, 35].map((week) => (
+            <Grid container alignItems="center" justifyContent="center">
+              {daysThisMonth.slice(week, week + 7).map((day) => {
+                const myDate = new Date(displayMonth.getFullYear(), displayMonth.getMonth(), day);
+                return (
+                  <Grid item xs={12 / 7} alignItems="center" justifyContent="center">
+                    <Box marginLeft={1.5} marginTop={2}>
+                      {day < week - 7 || day < 1 || day > week + 7 ? (
```

Comment:
make this boolean sequence a const, give it a name that reflects what's being checked, then call it here